.gitignore

```diff
@@ -2,4 +2,5 @@ node_modules
 saucelabs
 testingbot
 browserstack
+selenium-standalone
```

Comment:
Ah, for some reason I thought it might be a holdover from tests with the selenium-standalone module.